protobuf/test/example.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016 The Closure Rules Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+goog.provide('io.bazel.rules.closure.protobuf.Example');
+
+goog.require('proto.io.bazel.rules.closure.protobuf.Message');
+
+
+
+/**
+ * Example page.
+ * @param {string} field Value to set for "field".
+ * @constructor
+ * @final
+ */
+io.bazel.rules.closure.protobuf.Example = function(field) {
```

Comment:
Strangely enough, I did very little. I don't think I even had JS readability. After closure was open sourced I started using it heavily and learned a lot from reading the library and style guide as well as getting feedback from gjslint. Thanks.